google/devtools/clouderrorreporting/v1beta1/errorreporting_gapic.yaml

```diff
@@ -80,6 +80,10 @@ interfaces:
     init_fields:
     - project_name%project=$PROJECT_ID
     - event.message="[MESSAGE]"
+    - event.service_context.service="[SERVICE]"
+    - event.context.report_location.file_path="path/to/file.lang"
```

Comment:
No, I think you just need the fields set. It can just be the same for all languages.